### PR TITLE
feat(docs): Drawer code for composition

### PR DESCRIPTION
**Pull request by XingXiaoWu**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: website/docs/en-US/drawer.md [hunk 1]
```diff
@@ -51,6 +51,34 @@ Callout a temporary drawer, from multiple direction
...
+  import { defineComponent, ref, getCurrentInstance } from 'vue';
+
+  export default defineComponent({
+    setup() {
+      const { proxy } = getCurrentInstance();
+      const drawer = ref(false);
+      const direction = ref('rtl');
+      const handleClose = (done) => {
+        proxy
+          .$confirm('Are you sure you want to close this?')
```
Comment (iamkun) [inline website/docs/en-US/drawer.md:66]: Can we use 
```
import { ElMessageBox } from 'element-plus';
ElMessageBox.confirm()
```

to make it clear?

Comment (XingXiaoWu) [reply website/docs/en-US/drawer.md:66]: yeah, but I think it need to show how to use the ElMessage in vue instance.So I think it need to use `proxy`.

Comment (iamkun) [reply website/docs/en-US/drawer.md:66]: To know how to use Message in vue instance, our user could refer to the doc of Message component.

To us, it is better to use `ElMessageBox.confirm()` directlly.

Comment (XingXiaoWu) [reply website/docs/en-US/drawer.md:66]: Well, if you think that's better, I'll revise it later

Comment (iamkun) [reply website/docs/en-US/drawer.md:66]: Yes, please. Cheers

**Comment by XingXiaoWu** [comment]
@iamkun It has been updated. Thank you for your review

**Review by iamkun — APPROVED** [review]

**Comment by iamkun** [comment]
cool

**Pull request merged**